docarray/index/backends/hnswlib.py

```diff
@@ -167,21 +177,22 @@ class DBConfig(BaseDocIndex.DBConfig):
 
         work_dir: str = '.'
         default_column_config: Dict[Type, Dict[str, Any]] = field(
-            default_factory=lambda: {
-                np.ndarray: {
-                    'dim': -1,
-                    'index': True,  # if False, don't index at all
-                    'space': 'l2',  # 'l2', 'ip', 'cosine'
-                    'max_elements': 1024,
-                    'ef_construction': 200,
-                    'ef': 10,
-                    'M': 16,
-                    'allow_replace_deleted': True,
-                    'num_threads': 1,
+            default_factory=lambda: defaultdict(
+                dict,
+                {
+                    np.ndarray: {
+                        'dim': -1,
+                        'index': True,  # if False, don't index at all
+                        'space': 'l2',  # 'l2', 'ip', 'cosine'
+                        'max_elements': 1024,
+                        'ef_construction': 200,
+                        'ef': 10,
+                        'M': 16,
+                        'allow_replace_deleted': True,
+                        'num_threads': 1,
+                    },
                 },
-                # `None` is not a Type, but we allow it here anyway
-                None: {},  # type: ignore
```

Comment:
not really know why it was here, but why removing it?

Reply:
I think it's because python_type_to_db_type function was returning None for types other than vector, but I changed it with defaultdict which returns an empty dict by default